ffi/src/server.rs

```diff
@@ -301,8 +301,20 @@ pub fn start_os_proxy(
             Ok(OsProxy { server, conf_tx })
         })
     }
-    #[cfg(not(windows))]
+    #[cfg(target_os = "macos")]
+    {
+        copy_dir(
+            Path::new("mitmproxy_rs/MitmProxyAppleTunnel.app/"),
```

Comment:
This here won't work depending on your current working directory. Take a look at how we do it for Windows above with `py.import(...).filename()`. :)